images/macos/software-report/SoftwareReport.Toolcache.psm1

```diff
@@ -14,13 +14,15 @@ function Get-ToolcachePythonVersions {
 
 function Get-ToolcachePyPyVersions {
     $toolcachePath = Join-Path $env:HOME "hostedtoolcache/PyPy/*/x64"
-    Get-ChildItem -Path $toolcachePath | Sort-Object { [Version] $_.Parent.Name } | ForEach-Object {
+    $output = Get-ChildItem -Path $toolcachePath | Sort-Object { [Version] $_.Parent.Name } | ForEach-Object {
```

Comment:
nit: why do we need this change? looks like function will work the same way like before your changes?

Reply:
corrected